common-lib/src/cmr/common/util.clj

```diff
@@ -1039,12 +1009,12 @@
               (and (= "JWT" (:typ header-data))
                    (= "Earthdata Login" (:origin header-data)))
               false)
-            (catch com.fasterxml.jackson.core.JsonParseException e false))
+            (catch com.fasterxml.jackson.core.JsonParseException _e false))
           false))
       false)))
 
 ;; Note: Similar code exists at gov.nasa.echo.kernel.service.authentication
-(def URS_TOKEN_MAX_LENGTH 100)
+(def URS_TOKEN_MAX_LENGTH "Back in the day, this is how you knew it was URS" 100)
```

Comment:
funny!

Reply:
it's really hard coming up with doc strings after the fact